{"name": "test_get_all_repos", "status": "broken", "statusDetails": {"message": "NameError: name 'repos' is not defined", "trace": "def test_get_all_repos():\n        global repos\n        response = api.send_get(read_user_repos_endpoint(), auth=auth)\n        repos_list_response = Apiresponses(response)\n        if repos_list_response.get_status_code() == 200:\n            repos = repos_list_response.get_json()\n    \n            for repo in repos:\n                print(repo['name'])\n        else:\n            print(\"Oops, something went wrong\")\n    \n>       assert len(repos) > 0, \"This account not contains any repos\"\nE       NameError: name 'repos' is not defined\n\ntest_all_together.py:48: NameError"}, "attachments": [{"name": "stdout", "source": "934e4703-9054-4847-8562-c5c91c56890c-attachment.txt", "type": "text/plain"}], "start": 1679227478565, "stop": 1679227478831, "uuid": "d0e68fab-fc9a-4b76-8750-1edfa537f4c8", "historyId": "9b3036f8d234a0dabc5258b89f925022", "testCaseId": "6b2eab30c235eb72c8cf9b9b214466b0", "fullName": "test_all_together#test_get_all_repos", "labels": [{"name": "suite", "value": "test_all_together"}, {"name": "host", "value": "AlphaTest"}, {"name": "thread", "value": "13220-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_all_together"}]}